src/plugins/intel_cpu/tests/functional/shared_tests_instances/single_layer_tests/eltwise.cpp

```diff
@@ -80,7 +80,8 @@ std::vector<EltwiseTypes> eltwise_op_types = {
         EltwiseTypes::FLOOR_MOD,
         EltwiseTypes::SQUARED_DIFF,
         EltwiseTypes::POWER,
-        EltwiseTypes::MOD
+        EltwiseTypes::MOD,
+        EltwiseTypes::EQUAL_ELTWISE
```

Comment:
Please take a look at [my comment](https://github.com/openvinotoolkit/openvino/pull/30293#discussion_r2057640163) from your previous PR. This comment is still valid:

No need to add `EQUAL_ELTWISE` to eltwise enums. - `Equal` is tested [here](https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_cpu/tests/functional/shared_tests_instances/single_layer_tests/comparison.cpp) - you don't need to update anything in tests. Just to launch existing tests using `--gtest_filter="*smoke*Comparison*Equal*"`

Reply:
This comment is still valid :)

You removed `EQUAL_ELTWISE` from this list but there are sill changes in enum. Please remove all changes from tests including enum.